build/targets/KoreBuild.SolutionBuild.targets

```diff
@@ -148,7 +148,9 @@ solutions as well as the current file
   <Target Name="_GetRestoreItemsForSolutions"
     DependsOnTargets="_ResolveSolutions"
     BeforeTargets="_LoadRestoreGraphEntryPoints"
-    Condition="'@(_SolutionItems)' != '' AND '$(PreflightRestore)' != 'true'">
+    Condition=" '$(PreflightRestore)' != 'true' ">
+    <Error Text="No solutions found to build in '$(RepositoryRoot)'" Condition=" '@(_SolutionItems)' == '' " />
```

Comment:
What are the valid scenarios where `'$(PreflightRestore)' != 'true'`? And which MSBuild step do you mean?

What I'm trying to avoid is having to specify `.\build.cmd "/t:_ResolveSolutions;Restore". That and other target mixes that ensure `@(_SolutionItems)` has been set are what works today.